src/app/api/team-member/[memberId]/route.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import { NextResponse } from "next/server";
+
+const getATeamMember = async (req, res) => {
+  try {
+    const {data} = prisma.teamMember.findUnique({
+      where: {
+        id: req.query.memberId
+      }
+    });
+
+    return new NextResponse.json(data);
```

Comment:
```suggestion
    return NextResponse.json(data);
```

From what I know, we don't return a new Response generally. The typescript version of the above code gives an error if we return a new response + in acc to docs, we just return the Response.

![image](https://github.com/mlscvitpune/MLSC-VIT-PUNE-WEBSITE-2023/assets/92905626/9ef1da24-6d7b-42e7-b711-93785bc2b3b6)

If we don't return the instance then the error goes.